### PR TITLE
JR2-1229 Fix elastic search mapping and fields in search query

### PR DESCRIPTION
**Pull request by varusakov**

### CODE REVIEW

File: infrastructure-elasticsearch/src/main/resources/config/elasticsearch/mappings/api-user.json [hunk 1]
```diff
@@ -35,6 +42,7 @@
           },
           "companyEmail": {
             "type": "text",
+            "analyzer": "ascii_folding",
```
Comment (birom) [inline infrastructure-elasticsearch/src/main/resources/config/elasticsearch/mappings/api-user.json]: Why do we need ascii_folding for email?

Comment (varusakov) [reply infrastructure-elasticsearch/src/main/resources/config/elasticsearch/mappings/api-user.json]: for case insensitive matching

Comment (birom) [reply infrastructure-elasticsearch/src/main/resources/config/elasticsearch/mappings/api-user.json]: The standard analyser already has the Lower Case Token Filter.

Comment (varusakov) [reply infrastructure-elasticsearch/src/main/resources/config/elasticsearch/mappings/api-user.json]: Well, I need to check this

Comment (birom) [reply infrastructure-elasticsearch/src/main/resources/config/elasticsearch/mappings/api-user.json]: thx

Comment (varusakov) [reply infrastructure-elasticsearch/src/main/resources/config/elasticsearch/mappings/api-user.json]: Thank you @birom. Default analyzer contains lower case filter.

**Pull request merged**